SCI-FI_FAIR/presenter-npm-in-a-box.md

```diff
@@ -0,0 +1,17 @@
+# What is the NAME of your project?
+NPM in a box
+
+# What DOES THE PROJECT DO/what will you SHOW?
+_Briefly explain your madcap decentralized science experiment and why it will excite and inspire the fair attendees :)_
+
+# WHO will PRESENT the project?
+_PREFERRED NAME(S) and CONTACT EMAIL(S), please. The fair takes place 16:30 - 18:30 on Saturday, booked against the Unconference, so we recommend assigning 2-3 people in shifts, if possible. If you'll need to leave your post unattended for part of the time, please note that._ 
+
+# WHAT EQUIPMENT will you need?
+_The standard equipment is a table with a power strip. We can provide a MONITOR if needed (specify it now so we can rent one). OTHER EQUIPMENT or materials can also be provided within reason -- we want this fair to be distinctly more fun than the average conference poster session so creative execution ideas are welcome! Please include any ASSISTIVE DEVICES needed._
+
+# Do you have a LOGO/HERO IMAGE for the main projector?
+_Please link an image you'd like shown in the slideshow on the main projector in this Fair room (1280 x 720 or higher resolution image or 720p video loop/gif of <10sec)_
```

Comment:
I like this one:

![jen-npm](https://user-images.githubusercontent.com/665810/60212962-cde40300-9862-11e9-81e5-635da1273fe1.jpg)

But this one is probably better:

![npm-in-a-box](https://user-images.githubusercontent.com/665810/60212987-d5a3a780-9862-11e9-87c5-88c419f50411.png)

So, uh:

```suggestion
![npm-in-a-box](https://user-images.githubusercontent.com/665810/60212987-d5a3a780-9862-11e9-87c5-88c419f50411.png)
```